packages/block-editor/src/components/block-mover/index.native.js

```diff
@@ -20,48 +38,80 @@ const BlockMover = ( {
 	onMoveUp,
 	firstIndex,
 	rootClientId,
+	horizontalDirection,
 } ) => {
+	const {
+		firstButtonIcon,
+		secondButtonIcon,
+		firstButtonHint,
+		secondButtonHint,
+		firstBlockTitle,
+		lastBlockTitle,
+	} = horizontalDirection ? horizontalMover : verticalMover;
+
 	if ( isLocked || ( isFirst && isLast && ! rootClientId ) ) {
 		return null;
 	}
 
+	const getFirstMoverButtonTitle = () => {
+		const fromIndex = firstIndex + 1;
+		const toIndex = firstIndex;
+		return horizontalDirection
+			? sprintf(
+					/* translators: accessibility text. %1: current block position (number). %2: next block position (number) */
+					__( 'Move block left from position %1$s to position %2$s' ),
+					fromIndex,
+					toIndex
+			  )
+			: sprintf(
+					/* translators: accessibility text. %1: current block position (number). %2: next block position (number) */
+					__( 'Move block up from row %1$s to row %2$s' ),
+					fromIndex,
+					toIndex
+			  );
+	};
```

Comment:
Is it possible to move that string to `verticalMover` and `horizontalMover` and use it like this?
```suggestion
              const { firstMoverButtonTitle } = horizontalDirection ? horizontalMover : verticalMover;
              /* translators: accessibility text. %1: current block position (number). %2: next block position (number) */
		return  sprintf( __( firstMoverButtonTitle ), fromIndex, toIndex )
			
```

Reply:
I have pushed changes according to our conversation